inc/deprecated/3.5.php

```diff
@@ -1,6 +1,70 @@
 <?php
 defined( 'ABSPATH' ) || die( 'Cheatin&#8217; uh?' );
 
+/**
+ * Purge all the domain
+ *
+ * @since 2.6.8
+ *
+ * @param string $root The path of home cache file.
+ * @param string $lang The current lang to purge.
+ * @param string $url  The home url.
+ */
+function rocket_varnish_clean_domain( $root, $lang, $url ) {
+    rocket_varnish_http_purge( trailingslashit( $url ) . '?vregex' );
```

Comment:
For all functions in the file, missing `_deprecated_function()` and DocBlock tags.